lib/email_tracking_retriever.py

```diff
@@ -290,5 +294,15 @@ def log_in_if_necessary(driver: WebDriver):
   orders_containers = driver.find_elements_by_id('ordersContainer')
   if len(orders_containers) == 0:
     # likely a login screen
+    print("Signing into Amazon ...")
     driver.find_element_by_css_selector('form[name="signIn"]')
-    input('Please log in to an Amazon account on the opened Chrome profile. Hit ENTER when done.')
+    driver.find_element_by_xpath("//input[@type='email']").send_keys(AMAZON['email'])
+    driver.find_element_by_xpath("//input[@type='submit']").click()
+    time.sleep(2)
```

Comment:
What's up with the sleeps? Are they strictly necessary? I'd expect that they aren't.